src/ui/SettingsButton.tsx

```diff
@@ -26,42 +26,50 @@ export default function SettingsCard(
   const open = () => setIsOpen(true);
   const close = () => setIsOpen(false);
 
+  const buttonTextColor = useColorModeValue('ui.black', 'ui.white', 'ui.black');
+  const checkedButtonBgColor = useColorModeValue(
+    'ui.gray.light-warm',
+    'ui.gray.x-dark',
+    'ui.sepiaChecked'
+  );
+  const contentBgColor = useColorModeValue('ui.white', 'ui.black', 'ui.white');
+  const iconFill = useColorModeValue(
+    'ui.gray.icon',
+    'ui.white',
+    'ui.gray.icon'
+  );
   const paginationValue = props.state?.settings?.isScrolling
     ? 'scrolling'
     : 'paginated';
-  const contentBgColor = useColorModeValue('ui.white', 'ui.black', 'ui.white');
 
   return (
     <>
       <Popover
-        placement="bottom"
+        gutter={0}
+        placement="bottom-end"
         isOpen={isOpen}
         onOpen={open}
         onClose={close}
+        offset={[-200, 0]}
```

Comment:
@oliviawongnyc Let's try one more thing. Remove this offset and give the `PopoverContent`  a `width="inherit"` props. I think this fixes the overflow bug.

It seems like it could be an issue with the width: https://github.com/chakra-ui/chakra-ui/issues/4104#issuecomment-853764916

Reply:
Thank you, both! I merged in your PR @kristojorg. Looks great.